src/js/profile/mobile/widget/Calendar.js

```diff
@@ -298,6 +388,10 @@
 				element.appendChild(controllerElement);
 				element.appendChild(viewTableElement);
 
+				if (element.getAttribute("data-past-selection")) {
+					Calendar.pastSelection = element.getAttribute("data-past-selection");
```

Comment:
TAU supports the prototype._configure function to get / set the default options.
It would be good to use the '_configure' function

Reply:
I use '_configure' function.

done.